<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>result_form</class>
 <widget class="QWidget" name="result_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>350</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Result</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(34,40,49)</string>
  </property>
  <widget class="QFrame" name="result_frame">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>25</y>
     <width>250</width>
     <height>300</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(57, 62, 70);
border-radius: 20px;
padding: 5px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="Result_close">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>260</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK HK Black</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{
	background-color:rgb(0, 123, 131);
}
QPushButton{
	background-color:rgb(0, 173, 181);
	border-radius: 10px;
	padding: 5px;
}
</string>
    </property>
    <property name="text">
     <string>취소</string>
    </property>
   </widget>
   <widget class="QListWidget" name="Result_table">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>250</width>
      <height>250</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>250</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>250</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Umpush</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#00ADB5</string>
    </property>
    <property name="autoScrollMargin">
     <number>16</number>
    </property>
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="viewMode">
     <enum>QListView::ListMode</enum>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="selectionRectVisible">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
